### DISCUSSION TITLE
Add View in Backpack admin echo system without crud

### DISCUSSION THREAD

**Discussion by NeguinhoAdmin** [Laravel-Backpack/community-forum#1082] [General]
Hi ,

I need to add a view that might have iframe of my other admin app reside outside Laraval Backpack echo system. I need to add it as view or render within Backpack admin.

Can anyone tell me optimal way to do it, obviously, I'll make separate route for it.

Thank you

**Comment by jcastroa87** [comment]
Hello @NeguinhoAdmin 

You can create a custom operation to working with a custom view as our [docs](https://backpackforlaravel.com/docs/6.x/crud-operations#creating-a-new-operation-with-an-interface-1) says. In any case, is always tricky work with iframe, but this way will be an start point.

Cheers.

**Comment by jcastroa87** [comment]
Due to not activity i will close this issue, but please feel free to re-open or create a new one if needed.

Cheers.
